### PR TITLE
TM-3554: Update/panel meeting agenda routes

### PR DESCRIPTION
**Pull request by pveland621**
ACs:
1. Go Back button displays in new Panel Meeting Agenda page and returns user to Panel Meeting Search page
2. Using the Go Back button, make sure that filters on the Panel Meeting Search page persist (meeting type, status, etc.)
3. Verify URLs are correct for AO and CDO:
    
    profile/{cdo | ao}/panelmeetingagenda

Cory - I obviously used Panel Meeting Search as a template, and tried to remove the unnecessary code for the new page (mainly the filters). Let me know if I left any 'bloat' in there that's not applicable to the page

### CODE REVIEW

File: src/Components/Panel/PanelMeetingSearchRow/PanelMeetingSearchRow.jsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ const PanelMeetingSearchRow = ({ isCDO, result, showCreate }) => {
           {
             !!showCreate &&
             <div className="button-box-container">
-              <LinkButton className="button-box" toLink={`/profile/${userRole}/panel/`}>Go to Panel</LinkButton>
+              <LinkButton className="button-box" toLink={`/profile/${userRole}/panelmeetingagenda/`}>Go to Panel</LinkButton>
```
Comment (SidelineCory24) [inline src/Components/Panel/PanelMeetingSearchRow/PanelMeetingSearchRow.jsx:53]: 😎

**Comment by SidelineCory24** [comment]
@pveland621 thanks for the initial template setup for the search/filters. I named the folder on my end `PanelMeetingAgenda2` to avoid merge conflicts. Once this is merged I'll copy my work over to your files.

**Comment by pveland621** [comment]
ah my bad, should have double checked for merge conflicts, as the template setup wasn't technically part of the ticket

**Comment by SidelineCory24** [comment]
> ah my bad, should have double checked for merge conflicts, as the template setup wasn't technically part of the ticket

all good, I'd rather avoid the merge conflicts lol

**Comment by SidelineCory24** [comment]
Lets move the back button inbetween the header and the search bar, something like this. You'll likely need to move the margin from the button when it's on this page as well

image

**Comment by SidelineCory24** [comment]
> > Let's move the back button inbetween the header and the search bar, something like this. You'll likely need to move the margin from the button when it's on this page as well
> > image
> 
> gotcha, that was a result of a conversation with elizabeth and john, but i can revert it back to how it is in the screenshot

ah okay, you can leave it as is then

**Review by SidelineCory24 — CHANGES_REQUESTED** [review]
Looks good to go!🚀 Go back button just needs to be left at the top instead of in the middle. Great work!

**Review by SidelineCory24 — APPROVED** [review]

**Pull request merged**